.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+  # Dependencies listed in go.mod
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

Comment:
Do we fancy a day to check for new versions? Say Wednesday? 
The default behavior will make depandabot _check at random time on Monday_:
Referring: [schedule.day](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleday)

Reply:
I think random time on Monday is fine to start, we can tweak it later if needed